controller/ReviewTool.php

```diff
@@ -45,4 +46,12 @@ public function run()
             );
         }
     }
+
+    public function launch1p3(): void
+    {
+        $message = $this->getValidatedLtiMessagePayload();
+
+        LtiService::singleton()->startLti1p3Session($message);
```

Comment:
Please bump the taoLti extension version in `composer.json` dependencies, so that it points to the one, introducing that method.